pylint_odoo/misc.py

```diff
@@ -145,19 +145,27 @@ def _check_{NAME_KEY}(self, module_path)
                         self.add_message(msg_code, node=node,
                                          args=msg_args)
 
-    def filter_files_ext(self, fext, relpath=True):
+    def filter_files_ext(self, fext, relpath=True, skip_examples=True):
         """Filter files of odoo modules with a file extension.
         :param fext: Extension name of files to filter.
         :param relpath: Boolean to choose absolute path or relative path
                         If relpath is True then return relative paths
                         else return absolute paths
+        :param skip_examples: Boolean to skip "examples" folder
         :return: List of paths of files matched
                  with extension fext.
         """
+        dirnames_to_skip = ['example', 'examples', 'sample', 'samples', 'lib']
         if not fext.startswith('.'):
             fext = '.' + fext
         fext = fext.lower()
         fnames = self.ext_files.get(fext, [])
+        for fname in fnames:
+            dirnames = os.path.dirname(fname).split(os.sep)
+            for dirname_to_skip in dirnames_to_skip:
+                if dirname_to_skip in dirnames:
+                    fnames.remove(fname)
```

Comment:
@pedrobaeza 
I saw your point.
My point consider 1 remove without error.
But your point consider more than 1 remove with error.

You are right!
We could have more than 1 remove because we have 2 nested loops.

Thanks!